docs/user/OIDC-publishing/troubleshooting.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Troubleshooting
+---
+
+{{ preview('oidc-publishing') }}
+
+# Troubleshooting
+
+Here's a quick enumeration of errors you might see from the `mint-token`
+endpoint:
+
+* `not-enabled`: this indicates that PyPI's backend has
+  disabled OIDC entirely. You should not see this message during normal
+  operation for the duration of the OIDC beta, **unless** PyPI's admins
+  have decided to disable OIDC support.
+* `invalid-payload`: the OIDC token payload submitted to the `mint-token`
+  endpoint is not formatted correctly. The payload **must** be a JSON serialized
+  object, with the following layout:
+
+  ```json
+  {
+    "token": "oidc-token-here"
+  }
+  ```
+
+  No other layouts are supported.
+* `invalid-token`: the OIDC token itself is either formatted incorrectly,
```

Comment:
Maybe need an extra newline here to restart the bullets?

<img width="708" alt="image" src="https://user-images.githubusercontent.com/1324225/227345523-b3401281-03ba-4594-b52d-b147e0aa8207.png">


```suggestion

* `invalid-token`: the OIDC token itself is either formatted incorrectly,
```